packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -70,55 +97,71 @@ export class VaAccordion {
 
   @Listen('accordionItemToggled')
   itemToggledHandler(event: CustomEvent) {
-    // eslint-disable-next-line i18next/no-literal-string
-    const clickedItem = (event.target as HTMLElement).closest('va-accordion-item');
-    // Usage for slot to provide context to analytics for header and level
-    const header = clickedItem.querySelector('[slot="headline"]');
-    // using the slot to provide context to analytics for header and level
-    let headerText
-    let headerLevel
-    if (header) {
-      headerText = header?.innerHTML
-      headerLevel = parseInt(header?.tagName?.toLowerCase().split('')[1]);
-    } else {
-      // using the props to provide context to analytics for header and level
-      headerText = clickedItem.header
-      headerLevel = clickedItem.level
-    }
+    
 
-    if (this.openSingle) {
-      getSlottedNodes(this.el, 'va-accordion-item')
-        .filter(item => item !== clickedItem)
-        /* eslint-disable-next-line i18next/no-literal-string */
-        .forEach(item => (item as Element).setAttribute('open', 'false'));
-    }
+    /*if (this.uswds) {
+      if (this.openSingle) {
+        let items = getSlottedNodes(this.el, 'va-accordion-item');
+        items.forEach((item) => {
+          let el = item as Element;
+          let button = el.shadowRoot.querySelector('button.usa-accordion__button') as HTMLButtonElement;
+          if (button.getAttribute('aria-expanded') === 'true') {
+            button.click()
+          }
+        })
+      }
+    } else {*/
```

Comment:
Do we need all of this commented out code?

Reply:
Removed